docs/pages/box.js

```diff
@@ -404,10 +404,12 @@ function Example() {
 
       <MainSection name="Variants">
         <MainSection.Subsection
-          description={`Borders are controlled by the \`borderStyle\` property. Specifying a size ("sm" or "lg") enables a solid, light gray color in that width, while specifying "shadow" adds a box-shadow instead.`}
+          description={`Borders are controlled by the \`borderStyle\` property. Specifying a size ("sm" or "lg") enables a solid, light gray color in that width. Specifying "shadow" adds an even box-shadow around the entire container, while \`raisedTopShadow\` and \`raisedBottomShadow\` add shadows to indicate an elevated header or footer. See the [elevation foundations page](/elevation) for more details.`}
```

Comment:
nit 

Specifying \`shadow\`

![Screen Shot 2022-04-05 at 1 59 20 PM](https://user-images.githubusercontent.com/10593890/161820082-cfe3928e-4a7a-4cd0-ad69-dda79b9cb51d.png)

so all values are highlighted the same